src/types/Todo.ts

```diff
@@ -0,0 +1,6 @@
+export interface todo {
```

Comment:
```suggestion
export interface Todo {
```
it's better to name interfaces with upper camel case